### ISSUE TITLE
Is it possible to specify string filtering type like startWith or contains ?

### ISSUE THREAD

**Issue by noufionline** [StefH/GraphQL.EntityFrameworkCore.DynamicLinq#15]

**Comment by StefH** [comment]
Yes. See https://raw.githubusercontent.com/StefH/System.Linq.Dynamic.Core/master/test/System.Linq.Dynamic.Core.Tests/QueryableTests.Contains.cs

**Comment by noufionline** [comment]
@StefH 
Thanks a lot for your quick reply. 
What I am looking for is to have a string Contains and StartWith example.

Name.Contains("SomeText") OR Name.StartsWith("SomeText")
`
WHERE Name LIKE @Name 

`

**Comment by StefH** [comment]
Ah Sorry. I thought you asked this question for other project.

But this is not yet supported. Sorry.

**Comment by noufionline** [comment]
Is there any work around which can be done like combination of your current solution with additional customisation to get the this supported?

**Comment by StefH** [comment]
I am currently also building better nested query functionality. However, I am a bit stuck on that part.

Adding new search functionality is possible.

But first I need to get that new functionality working. Or at least get the code working again.
